plugins/out_stackdriver/stackdriver.c

```diff
@@ -546,22 +611,41 @@ static int stackdriver_format(const void *data, size_t bytes,
          *  "timestamp": "..."
          * }
          */
+        
+        /*  Parse jsonPayload and extract operation first*/
+        operation_id = flb_sds_create("");
+        operation_producer = flb_sds_create("");
+        operation_existed = extract_operation(&operation_id, &operation_producer,
```

Comment:
how about `operation_extracted`? operation may exist but not comply with the format, so I think calling it `operation_extracted` would make it more clear. What do you think?

Reply:
I have renamed 'operation_existed' as 'operation_extracted'.